src/components/Numbers/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import dotsImage from './img/dots.png';
+import circleImage from './img/circle.png';
+import { charityAPI } from '../../clients';
+import './style.css';
+
+const Number = ({ number, title }) => {
+  return (
+    <div className="statistics-content__item justify-end flex flex-col w-1/4">
+      <span className="statistics-content__item__value text-center tracking-wide text-c200 text-xl font-light font-body leading-loose">
+        {number}
+      </span>
+      <h3 className="statistics-content__item__name mt-3 tracking-wide capitalize font-light text-c100 text-md">
+        {title}
+      </h3>
+    </div>
+  );
+};
+
+class Numbers extends Component {
+  state = {
+    data: {
+      backgroundImage:
+        'https://lh3.googleusercontent.com/qWM9ByJ3Fbz2tPnnbF4paUtl3N5gRCG01DFESqYOoM9tJIHx4fdGH4ccHX6mwPSfzPdvEiNZUfs2m1fNt_msn02AjDAROWbMmLkYPgOZOs5IEt09BtiBNBatgMQ9Hg-qjEIDP5Pt6WY4uImsKrRrm724Kqx0M5ka-bn4k0n_vjQ0hVSDoxr5TYBTixHvuJbFEwvMOjWEYzri_xuKGhgAcHRzu1Rl70e2nwwJUy_34tgn1hC64Y0wzXhrw1Vq9CM32e0ilvoCAUCtPzCO_i187FgFX-_TZ7yJmb4Ry16XblLACFyRz7NzA25YGs264i2xTtH6ZlhsreofKoirHmLE2fMZkp7fq5MzfsBnLkgZpEPmBrL7HXf35L3VXbBN4j3XYypI9sJAFZTMJ0Dbv0dpWOMT1FebLEZrgz3zrAblD5kL1yljUJMPvfkrV8AYSyHfyBiHNAs0T6b4M5q406VH_ovmZPS_i1js3ZXyuC3H-nlrfUKJGlLU_9suRefWbaxVHEDRJdDG6aEgBLKakc231ZMO51ixjdoH34w7I0YIY8Y6n5j8yedFIRTjfaBqcXiDoaEF4cWjfORKmTSty8qBOHoC2Bt_mj9COHOXD6gsoTUSb3BfL0m7Vz4NG1sLipok5V2-cOY8vHZAZ6kAKXGfppqOsrn6MNlA0Ls_S7Oz0DHp3hMJSfmaJ6nkS4AHOhNe9YdWbMpddht_7IYewcUfjI1YWRIAjwn-nyfHp0Nakvbb7CNpLGGL=w1354-h903-no'
+    },
+    loading: true,
+    error: false,
+    errorMessage: ''
+  };
+
+  componentDidMount() {
+    this._getData();
+  }
+
+  _getData = () => {
+    this.setState({ loading: true });
+    charityAPI('/speaking-numbers')
+      .then(({ data: { speaking_numbers, image_background } }) => {
+        this.setState({
+          loading: false,
+          data: {
+            numbers: speaking_numbers,
```

Comment:
You have changed the state here but `numbers` state was not initialized inside the state object. I think it is good practice to initialize the sate. 👍

Reply:
Yes, I agree with you.
But it will NOT cause an error since the initial state of `loading` is set to `true`, though I will initialize it with empty array.